app/assets/javascripts/admin/enterprises/controllers/enterprise_controller.js.coffee

```diff
@@ -21,6 +21,26 @@ angular.module("admin.enterprises")
     $scope.cancel = (destination) ->
       $window.location = destination
 
+    $scope.removeLogo = ->
```

Comment:
These methods are not the most important on this file so I'd move them further down on the file, I'd put them at the end of the file actually.